Seneca-PlaNA/body-contouring-clinic
# [Schedule] CRUD, and the calendar view

### How to test this PR

1. Go to `/Staff/Schedules`, you should be able to see the schedule list
2. You should also be able to see the details of schedule by click the detail button
3. You should also be able to delete and edit the schedule
4. Click "Schedule Calendar" in sidebar, you should be able to view all schedules in **day view** and **week view**
5. In **day view** calendar, you should be able to see the date changes when you click next or previous arrows 

Fixes: #213 
Fixes: #210 
Fixes: #166
